drivers/serial/Kconfig.xlnx

```diff
@@ -0,0 +1,14 @@
+# Kconfig - Xilinx UART configuration
+#
+# Copyright (c) 2018 Xilinx Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig UART_XLNX_PS
+	bool "Xilinx serial driver for MPSoC"
+	depends on SOC_XILINX_ZYNQMP
+	select SERIAL_HAS_DRIVER
+	help
+	  This option enables the UART driver for Xilinx MPSoC platforms.
+	  Say y if you wish to use serial port on Xilinx MPSoC platforms.
```

Comment:
Not sure if you really need the last line

Reply:
fixed